File: C/1165.c
#include <stdio.h>

main(){
int num,i=0,x,cont,div=0;
  scanf("%d", &cont);
  
  for(i=1; i<=cont; i++){
    scanf("%d",&num);
    
    for(x=1,div=0; x<=num; x++){
      
      if(num%x==0){
          div++;
      }
    }
      
      if(div==2){
          printf("%d eh primo\n", num);
      }
      
      else if(div>2){
            printf("%d nao eh primo\n", num);
      }
  }
return 0;
}

File: Ruby/1165.rb
i = 0
cont = gets.chomp.to_i

while i < cont
    num = gets.chomp.to_i
    div = 0
    x=1
    
    while x <= num
        if num % x == 0
            div = div + 1
        end
        x = x + 1
    end
    
    if div > 2
        puts "#{num} nao eh primo"
    else 
        puts "#{num} eh primo"
    end
    i = i + 1
end